README.md

```diff
@@ -49,6 +53,46 @@ banana-i18n is a javascript internationalization library that uses "banana" form
 ```
 npm i banana-i18n
 ```
+## Usage
+
+### Node.js
+
+```js
+const Banana = require('banana-i18n');
+
+// Initialize Banana-i18n with the default locale and messages
+const banana = new Banana('en', {
+    messages: {
+        'greet-user': 'Hello, $1!',
+        'items-count': '$1 item(s) found.',
+    }
+});
+
+// Translating a simple message
+console.log(banana.i18n('greet-user', 'Alice')); // Output: Hello, Alice!
+
+// Translating with variables
+console.log(banana.i18n('items-count', 5)); // Output: 5 item(s) found.
+
+// Changing the language dynamically
+banana.setLocale('fr');
+banana.load({
+    'greet-user': 'Bonjour, $1!',
+    'items-count': '$1 article(s) trouvé(s).',
+});
+
+console.log(banana.i18n('greet-user', 'Alice')); // Output: Bonjour, Alice!
+```
+
+### JavaScript
+
+For usage in JavaScript, please check the files under the `demo` folder after building the distribution files by running `npm run build`.
+
+## Demo
+
+See the library in action: https://wikimedia.github.io/banana-i18n/demo/
```

Comment:
the demo page shows nothing to me, is this normal?
<img width="715" alt="image" src="https://github.com/user-attachments/assets/884b8b9e-cbfe-4088-bad0-ec4f9492dbd6">

Reply:
Its trying to load the library from the dist folder but we don't push the files there. We have two options:

1. Remove the dist folder from `gitignore` and add the files from the build into the dist folder.
2. Add the build files separately under the demo.

Reply:
2. You can do it through the CI and only build and store it on gh-pages branch

Reply:
@hueitan That was something I was considering. I assume that would be easy to setup?

Reply:
> @hueitan That was something I was considering. I assume that would be easy to setup?

yes, we did that in wikipedia-preview, https://github.com/wikimedia/wikipedia-preview/blob/9d775e477919f4c3b17fb126685433e72baa2784/.github/workflows/main.yml#L26-L52, but through the GitHub Action.

Reply:
gh-pages branch approach seems better to me

Reply:
the demo page is available with this CI update https://github.com/wikimedia/banana-i18n/pull/88